python/semantic_kernel/planners/function_calling_stepwise_planner/function_calling_stepwise_planner.py

```diff
@@ -106,6 +112,9 @@ async def invoke(
         if not question:
             raise PlannerInvalidConfigurationError("Input question cannot be empty")
 
+        if not arguments:
```

Comment:
Check `if arguments is not None` instead, this will return false for a Kernel args object without values but with execution settings, I've been changing that in some other places need to check all...

Reply:
I actually want the check as `if argument is None`, then I want to create the KernelArguments with any **kwargs. By using `if arguments is not None` then we will skip the creation of the Kernel args -- other methods except them, even if they're just empty `{}`.

Reply:
I can make any adjustment in a next iteration, as needed.